Project/Assets/ML-Agents/Examples/Crawler/Prefabs/DynamicPlatform.prefab

```diff
@@ -540,6 +540,11 @@ PrefabInstance:
       propertyPath: targetToLookAt
       value: 
       objectReference: {fileID: 7802320107249901494}
+    - target: {fileID: 4622120667686875944, guid: 0456c89e8c9c243d595b039fe7aa0bf9,
```

Comment:
Hmm, don't think I meant to override this. Now I need to double check the model settings